python/lsst/ap/association/transformDiaSourceCatalog.py

```diff
@@ -378,12 +378,13 @@ def unpack(self, input_flag_values, flag_name):
         output_flags : `numpy.ndarray`
             Numpy named tuple of booleans.
         """
-        bit_names_types = self.output_flag_columns[flag_name]
-        output_flags = np.zeros(len(input_flag_values), dtype=bit_names_types)
+        output_flags = np.zeros(len(input_flag_values),
+                                dtype=[(name, bool) for name in self.output_flag_columns[flag_name]])
```

Comment:
I figured out what this does after falling down a `dtype` rabbit hole, but, possibly silly question, aren't they all required (by virtue of being flags) to just be `bool`? Is there a reason we bend over backwards to assign the dtypes one at a time?

Reply:
I have no idea why Chris originally made this a dict of tuples of `(name, bool)`, because they're all booleans because this is a bit field. That's why I changed that part of it.

I put in a dtype for the numpy array, so that it is a structured array with column names, each one a bool.